service/slice_config_webhook_validation.go

```diff
@@ -247,7 +247,16 @@ func preventUpdate(ctx context.Context, sc *controllerv1alpha1.SliceConfig) *fie
 }
 
 // validateQosProfile is a function to validate the Qos(quality of service)profile of slice
-func validateQosProfile(sliceConfig *controllerv1alpha1.SliceConfig) *field.Error {
+func validateQosProfile(ctx context.Context, sliceConfig *controllerv1alpha1.SliceConfig) *field.Error {
+	if len(sliceConfig.Spec.StandardQosProfileName) > 0 && len(sliceConfig.Spec.QosProfileDetails.QueueType) > 0 {
+		return field.Invalid(field.NewPath("Spec").Child("StandardQosProfileName"), sliceConfig.Spec.StandardQosProfileName, "StandardQosProfileName cannot be set when QosProfileDetails is set")
+	}
+	if len(sliceConfig.Spec.StandardQosProfileName) <= 0 && len(sliceConfig.Spec.QosProfileDetails.QueueType) <= 0 {
+		return field.Invalid(field.NewPath("Spec").Child("StandardQosProfileName"), sliceConfig.Spec.StandardQosProfileName, "Either StandardQosProfileName or QosProfileDetails is required")
+	}
+	if len(sliceConfig.Spec.StandardQosProfileName) > 0 {
+		existsQosConfigFromStandardQosProfileName(ctx, sliceConfig.Namespace, sliceConfig.Spec.StandardQosProfileName)
```

Comment:
We need to throw an error if not found, right?

Reply:
oh. missed that.